suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -482,8 +482,48 @@ export const selectIsPendingTransportEvent = (state: DeviceRootState) =>
 
 export const selectDevice = (state: DeviceRootState) => state.device.selectedDevice;
 
+export const selectIsDeviceUnlocked = (state: DeviceRootState) =>
+    !!state.device.selectedDevice?.features?.unlocked;
+
 export const selectDeviceState = (state: DeviceRootState) => state.device.selectedDevice?.state;
 
+export const selectDeviceType = (state: DeviceRootState) => state.device.selectedDevice?.type;
+
+export const selectIsDeviceProtectedByPin = (state: DeviceRootState) =>
+    !!state.device.selectedDevice?.features?.pin_protection;
```

Comment:
nit: Since there is a new `selectDeviceFeatures` selector, use it here.

Reply:
https://github.com/trezor/trezor-suite/pull/9576/commits/a28af4bc7cfec7a3476cfe5c4206fd65afbcdb7f